tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/util/ProjectImportUtil.java

```diff
@@ -500,6 +500,63 @@ else if( tempRecord.isTheme() )
         return isValid;
     }
 
+    public static IStatus validateSDKPath(final String currentPath)
+    {
+        IStatus retVal = Status.OK_STATUS;
+
+        if( !org.eclipse.core.runtime.Path.EMPTY.isValidPath( currentPath ) )
+        {
+            retVal = ProjectCore.createErrorStatus( "\"" + currentPath + "\" is not a valid path." );
+        }
+        else
+        {
+            IPath osPath = org.eclipse.core.runtime.Path.fromOSString( currentPath );
+
+            if( !osPath.toFile().isAbsolute() )
+            {
+                retVal = ProjectCore.createErrorStatus( "\"" + currentPath + "\" is not an absolute path." );
```

Comment:
1).The sdk absolute path checking just check whether the sdk location user typed can be found in  disk. It just check the sdk folder whether it can be found.
2).About the app.server.parent.dir value will be resolved by ant, It will be resolve and validate later. Before the ant resolve app.server.parent.dir, it can be relative path, ant resolve will help us to get the server absoulte path.